spec/models/user_spec.rb

```diff
@@ -217,6 +217,16 @@
         end
       end
     end
+
+    describe '#active_legacy_profile' do
+      it 'returns the active_legacy_profile' do
+        user = create(:user, :fully_registered)
+        profile1 = create(:profile, :active, :verified, user: user, idv_level: :legacy_unsupervised)
```

Comment:
Can we get a version of this test that does the same check for `in_person` and `legacy_in_person` just so we have the other possible values for `idv_level` covered